### ISSUE TITLE
Mappings - Volume:

### ISSUE THREAD

**Issue by jeremyf** [scientist-softserv/adventist-dl#188]
> Volume\_number and volume in OAI\_ADL are not mapping to Hyku (see line 38 of the mappings spreadsheet). SDAPI example: [OAI\_ADL](https://www.google.com/url?q=http://oai.adventistdigitallibrary.org/OAI-script?verb%3DGetRecord%26metadataPrefix%3Doai_adl%26identifier%3D13129455&sa=D&source=editors&ust=1671480230561230&usg=AOvVaw1DJ7c6yQLq36FS5B2zIPzq) and [Hyku](https://www.google.com/url?q=https://sdapi.b2.adventistdigitallibrary.org/concern/journal_articles/13129455_13th_sabbath_offering_northern_europe_west_africa?locale%3Den&sa=D&source=editors&ust=1671480230561478&usg=AOvVaw2KL7iyvEZSIzoW5rqtK2ji).  See also an ADL example: [OAI\_ADL](https://www.google.com/url?q=http://oai.adventistdigitallibrary.org/OAI-script?verb%3DGetRecord%26metadataPrefix%3Doai_adl%26identifier%3D22220536&sa=D&source=editors&ust=1671480230561705&usg=AOvVaw1yGW0rjanm_6px8Qdv9DAK) and [Hyku](https://www.google.com/url?q=https://adl.b2.adventistdigitallibrary.org/concern/published_works/22220536_sabbath_readings_for_the_home_circle_vol_2_moral_and_religious?locale%3Den&sa=D&source=editors&ust=1671480230561944&usg=AOvVaw2dfniadb0t19_g7PqwYihf).

From https://docs.google.com/document/d/1mIOT23UAilSO77pAlXYSWJEHw3YK3BNNQVTKNzd41ao/edit#

**Comment by KatharineV** [comment]
Can confirm this is happening on SDAPI and ADL both. The example above is SDAPI. Here is an example from ADL where volume and issue number don't transfer from OAI_ADL to Hyku: [MODS](http://oai.adventistdigitallibrary.org/OAI-script?verb=GetRecord&metadataPrefix=mods&identifier=22252214), [OAI_ADL](http://oai.adventistdigitallibrary.org/OAI-script?verb=GetRecord&metadataPrefix=oai_adl&identifier=22252214), [Hyku](https://adl.b2.adventistdigitallibrary.org/concern/published_works/22252214_prophetic_expositor_and_bible_advocate?locale=en).

**Comment by ShanaLMoore** [comment]
On Hold until client can unblock us. The collection sdapi:article may need to have an ark_id. The import is not successful at its current state. We are unable to view the final imported object, but the raw_metadata and parsed_metadata looks correct.

see [#collections_created?](https://github.com/samvera-labs/bulkrax/blob/35970cee7b1e48d00c3fc838d2ca5790c09e2c9c/app/models/bulkrax/oai_entry.rb#L51) from Bulkrax.

[Exhausted timeout](https://sentry.io/organizations/scientist-inc/issues/3827889423/?project=6745020&referrer=project-issue-stream) error in sidekiq.

Jeremy to add this to the client iteration meeting.

**Comment by jeremyf** [comment]
This is unblocked; our decision is to create collections as we encounter them in the OAI feed. I amended the testing instructions to now import 30 instead of 1.

**Comment by ShanaLMoore** [comment]
blocked, staging is busted

**Comment by ShanaLMoore** [comment]
unblocked, it appears this can be tested using the sdapi tenant

**Comment by DiemBTran** [comment]
**This passes SoftServ QA:** See attached screenshots.

tested on:
- importer: https://sdapi.s2.adventistdigitallibrary.org/importers/23?locale=en
- random work's entry on importer: https://sdapi.s2.adventistdigitallibrary.org/importers/23/entries/592?locale=en
- ^that same work's show page: https://sdapi.s2.adventistdigitallibrary.org/concern/journal_articles/13128989__1_million_bible_ministry_makes_major_impact?locale=en

<details><summary>Work show page shows "Volume" metadata</summary>


![Journal Article $1 million Bible ministry makes major impact ID 13128989__1_million_bible_ministry_makes_major_impact Hyku 2023-04-04 at 2 49 12 PM](https://user-images.githubusercontent.com/29311858/229930992-c149f54d-717e-41f8-9303-8fcc34ffd25a.jpg)</details>

**Comment by KatharineV** [comment]
Working as expected on SDAPI staging! Tested using this [importer](https://sdapi.s2.adventistdigitallibrary.org/importers/27?locale=en).
